languages/go/types/polar_types.go

```diff
@@ -1039,132 +1039,82 @@ func (result *Operator) UnmarshalJSON(b []byte) error {
 }
 
 func (variant Operator) MarshalJSON() ([]byte, error) {
-	switch inner := variant.OperatorVariant.(type) {
+	switch variant.OperatorVariant.(type) {
 
 	case OperatorDebug:
-		return json.Marshal(map[string]OperatorDebug{
-			"Debug": inner,
-		})
+		return json.Marshal("Debug")
```

Comment:
I don't understand what this was even doing before

Reply:
It was just wrong, Instead of serializing the ops as a string, eg `{"Expression": {"Operator": "Eq", args:  [...]}` It was serializing the operations like this  `{"Expression": {"Operator": {"Debug": {}}, args:  [...]}`